docs/comment_copy_guidelines.md

```diff
@@ -0,0 +1,133 @@
+_what_ to comment. This document talks mostly about _how_ to comment.
+
+## Working directory for contents
+
+Comments for the analyzer live in the [`exercism/website-copy`][git-website-copy]
+repository. Changes should be made via a PR on this repository, only changing
+the comments for one analyzer at a time. These will get the
+[`type/analyzer-comments` label][git-website-copy-label].
+
+### Generic comments
+
+Some comments are language specific and not exercise specific. These should be
+extracted into a generic comments file, in order to not duplicate commentary,
+which would lead to either outdated commentary or shotgun surgery when a change
+is made.
+
+## CI
+
+Because the comments don't live in the same repository as the analyzer, each
+analyzer should have CI that checks if the comments used in that specific
+analyzer (those that can become output), are comments on the `master` branch, on 
+the  [`exercism/website-copy`][git-website-copy] repository.
+
+At the moment of writing, [this issue][issue-ci-comments] tracks the status of any
+generalisation of this CI, if any.
+
+## Wording
+
+- Avoid unnecessarily wordy comments. Be concise.
+- Be neutrally observational; avoiding charged statements and blanket statements.
+- Make the recommendation explicit.
+- When possible, put the recommendation first, then the explanation.
+- Avoid "me", "I", "we", etc, since the bot is not a person.
+- Avoid "you" and "your code", because it can sometimes come across as judging 
+  the person rather than the code.
+- Avoid words like "just", "simply", "obviously", which can come across as 
+  condescending: if the comment is necessary, it was _obviously_ not _obvious_.
+- Avoid making assumptions about what people know and don't know. The only
+  exception is knowledge from already completed _core_ exercises. Avoid
+  "as you know", "as you remember", "as you learned", "now that we all
+  understand x", because even though something was said, the person doesn't 
+  necessarily understand it.
+
+### Guidance
+
+For the first few exercises of a track, the following is _extra_ important:
+
+- **Name the difference** between what the person is doing, and what is
+  'idiomatic' in the language.
+- **Use the proper terms** and nomenclature, so people can recognize those
+  concepts elsewhere, and also can research them by themselves.
+- **Keep it relatively short**, avoiding a wall of text or overwhelming them
+  with advice. If they have a great experience in the first exercise, they'll
+  come back and you'll have many more opportunities to provide feedback on all
+  of the things that you noticed.
+- **Don't give the solution** as general advice for all Exercism mentoring. 
+  Learning sticks when people discover the answer for themselves. That's a 
+  tremendously exhilarating experience, and the emotional kick makes it 
+  memorable. However, if the discovery **doesn't** trigger a dopamine hit, it 
+  makes total sense to show what it looks like.
```

Comment:
```suggestion
  makes total sense to show what it looks like. For example, we might choose to provide a small improvement on an approved solution will be a lot less exciting than giving someone a learning point on a solution that is being disapproved and therefore may warrant an example rather than a link.
```